lib/context.js

```diff
@@ -1,13 +1,17 @@
 import { readFile, writeFile } from 'mz/fs'
 import { resolve } from 'path'
 import { homedir } from 'os'
+import { sync } from 'find-up'
```

Comment:
sync might cause confusion here. Better import the whole `find-up` and call `findUp.sync`

Reply:
i was thinking this 👍

Reply:
or `import { sync as findUpSync } from 'find-up'`